tests/conftest.py

```diff
@@ -29,24 +29,19 @@
 
 # Multi modal related
 # You can use `.buildkite/download-images.sh` to download the assets
-_PIXEL_VALUES_FILES = [
+PIXEL_VALUES_FILES = [
```

Comment:
It seems to me as a convention that variables in this file should start with `_`, why did we change this?

Reply:
Other test files may need to import this to know how many image files to create the prompts for.

Reply:
Shouldn't we make fixtures out of them instead of importing them directly?

Reply:
Since there is no I/O involved with just declaring the filepaths, I think there is no need to use fixtures. I prefer being more explicit about where the variables are coming from.

Reply:
Yea I think this is fine then.